### ISSUE TITLE
Add some kind of failure reporting when XML writing fallback is used

### ISSUE THREAD

**Issue by neonichu** [CocoaPods/Xcodeproj#252]
While it seems the fallback is working as expected when ASCII plist writing cannot be done, we'd still want to diagnose those user problems.

**Comment by orta** [comment]
Is xml writing the pre-0.36 style?

**Comment by neonichu** [comment]
@orta Yep, indeed. Xcodeproj will fall back to it if it'll fail to use DevToolsCore.framework for some reason (can't find Xcode, can't find symbols, failure result, ...)

**Comment by orta** [comment]
Would it make sense to output something to verbose about using this, and maybe including it in a crash report?
